### DISCUSSION TITLE
Gogrepoc stop functioning normally and opens VS Code every time.

### DISCUSSION THREAD

**Discussion by trueBluewolf** [Kalanyr/gogrepo#118] [Q&A] [answered]
![Screenshot 2024-12-18 011939](https://github.com/user-attachments/assets/71fcb4fa-f624-44fa-9844-cafa32fff3d7)

Hi, sorry if this is a simple fix but why is gogrepoc doing this now. It also opens up VS Code afterwards every time and I can't get it to do anything.

**Comment by Kalanyr** [comment]
You've changed the execution association of the .py file type somehow I
think.

Reinstalling Python is probably the easiest way to fix this if you're
python environment is relatively minimal.

**Reply by trueBluewolf** [reply]
Didn't seem to work but found another solution I guess but its slightly annoying. Haven't learned python in a long time, but reinstalled python and the other libraries you have in the github again but it didn't work until I just said "python gogrepoc.py". Got it to work again but need to add 'python' every time now at the beginning

**Comment by Kalanyr** [comment, marked as answer]
Hmmmm. Interesting.  Reinstalling Python should have reassociated .py with
the python files so maybe it's a path problem?

What happens if you use "py GOGrepoc.py"  ?

**Reply by trueBluewolf** [reply]
That works too

**Reply by trueBluewolf** [reply]
I could be dumb... I think I just totally forgot about adding python or py first since im not use to python lol

**Comment by Kalanyr** [comment]
No, you're right , it should setup .py to be executable IIRC , I'm just not
sure why it's not. Unfortunately I've just gotten so used to using py to
manage my multiple Python versions that I don't recall how any of that
stuff works anymore there's probably a checkbox in the installer or a
command you can run in windows but it's long since escaped me.
